### DISCUSSION TITLE
Add an option for range of years

### DISCUSSION THREAD

**Discussion by projetos-sidnei** [onesine/react-tailwindcss-datepicker#93] [Ideas]
Hello, I'm not fluent in English, so I'm using a translator, I'd like to consider an implementation to limit the range of years, currently the component allows negative years and if I need to limit an initial and final year I can't, or there is some way that I did not find it.

![image](https://user-images.githubusercontent.com/59892360/225907238-12a60140-45c7-43dd-94d1-c2de06f2e5fa.png)

**Comment by onesine** [comment]
Hi @projetos-sidnei  👋 

Thanks for your feedback.
Sorry for the delay.

Indeed that would be interesting.

So we should:

- by default block the negative years.
- propose a year limiter for both calendars.
